docs/integration/kubernetes.md

```diff
@@ -0,0 +1,257 @@
+Kubernetes is a portable, extensible, open-source platform for managing containerized workloads and services, that facilitates both declarative configuration and automation. Because of its ever-growing popularity, we’ve decided to create a simple yet powerful Kubernetes monitoring integration that is easy to set up. You only need to install a tiny agent that adds basically no CPU or memory overhead. 
```

Comment:
I think we don't need things like "Because of its ever-growing popularity, we’ve decided to create a simple yet powerful Kubernetes monitoring integration that is easy to set up. You only need to install a tiny agent that adds basically no CPU or memory overhead. " because that's more like marketing lingo and not info that a Sematext user looking for how-to info needs to read.
n.b. some of our older docs pages suffer from this mixing of promo-style vs. how-to content, too.

Reply:
This can be changed to:

Kubernetes is a portable, extensible, open-source platform for managing containerized workloads and services, that facilitates both declarative configuration and automation. To start monitoring Kubernetes with Sematext, you only need to install a tiny agent that adds basically no CPU or memory overhead.